### ISSUE TITLE
Remove unused labels.

### ISSUE THREAD

**Issue by Carreau** [ipython/ipython#9780]
Can we get rid of un-used labels? At least the one that are useless on **closed** issues. We have several that are applied on 0 open issue. 

There are a few that we should keep (like `Closed-PR`), but many like backport-* docker-* that I would like to remove/replace by more explicit ones. 

cc @fperez , @ellisonbg , @willingc , @JamiesHQ , @minrk , @takluyver

**Comment by fperez** [comment]
no objection from me.

**Comment by minrk** [comment]
The backport ones might be useful for historical purposes, since they are the only mechanism of identifying backported PRs for the releases prior to milestones. I think there are lists in the release notes, so it's not critical if there's a reason to delete that information.

**Comment by fperez** [comment]
Good point about the backport ones... Overly long label lists do create clutter, though, so I guess it's a matter of balance: having the label makes it easy to find that info at any time right here, otherwise we have to go dig into the release notes. And while that is ok for manual inspection, it's harder for any kind of automated analysis that someone might want to try over our label usage.

Mmh... I could argue myself both ways on this one :)  I'm still leaning towards helping daily usage by removing clutter, but the destruction of useful data worries me. Given my recent screwup deleting the old nbconvert repo, that topic is a bit of a sore spot.

**Comment by Carreau** [comment]
> Mmh... I could argue myself both ways on this one :) I'm still leaning towards helping daily usage by removing clutter, but the destruction of useful data worries me. Given my recent screwup deleting the old nbconvert repo, that topic is a bit of a sore spot.

That's why I opened the issue.

The backport one is relatively easy, we can create milestones a-posteriori, and actually some of these already have a milestone.

They also get crosslink by the backport message. EG:

```
minrk added a commit that referenced this pull request on Apr 8, 2014
 @minrk	Backport PR #5459: Fix interact animation page jump FF  …
```

**Comment by Carreau** [comment]
Updated first comment to list the label that have only few issues. The backports represent only a fifth. So let's not consider those for removal. But things like Docker that have a total of 2 issues (1 open , 1 closed) for example.

**Comment by minrk** [comment]
@Carreau creating milestones from the labels sounds good to me, then deleting them would definitely be fine. We also made links to each backported issue/PR in the release notes, so it's not exactly lost information. Also +1 to removing the docker labels.

**Comment by willingc** [comment]
+1 to removing all suggested by @Carreau.

**Comment by fperez** [comment]
milestones sounds like a great solution, as they would be closed so they don't clutter the day-to-day milestones menu.

with that, +1 to the cleanup plan

**Comment by Carreau** [comment]
> milestones sounds like a great solution, as they would be closed so they don't clutter the day-to-day milestones menu.
> 
> with that, +1 to the cleanup plan

Ok i'll try to cleanup things, and leave backport lasts, and check milestones as well.
